### DISCUSSION TITLE
How long will it take one cat to catch one mouse?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#442] [Galaxy Trivia] [answered]
If five cats can catch five mice in five minutes.

**Comment by ParanoidUser** [comment]
@trevorwang, @skzidan, @wfrsk maybe? 😼

**Comment by c8f888391789da0901167ee6134c4eb2** [comment, marked as answer]
5 cats -> 5 mice -> 5 minutes

One cat instead of five takes five times more.

1 cat -> 5 mice -> 25 minutes

1 cat -> 1 mouse -> 5 minutes

**Comment by peterbaccaro** [comment]
**5 minutes**.
